### DISCUSSION TITLE
why is an underscore displaying in element name?

### DISCUSSION THREAD

**Discussion by awootton1** [FirelyTeam/firely-net-sdk#2030] [General]
In the diagnostic report resource, when the IssuedElement is null...and the IssuedElement.Extension is set to the data absent reason string, I am seeing an underscore in the element name when I do my POSTMAN POST.    

![underscore](https://user-images.githubusercontent.com/102695730/160983552-a5f84b73-30a3-455f-9447-76a701b80c8e.jpg)

**Comment by mbaltus** [comment]
This is how FHIR has specified to put an extension on a primitive data element in JSON. See http://hl7.org/fhir/json.html#primitive for more information.
